TestAssets/TestProjects/PerformanceTestProjects/SingleTargetGraph/SingleTargetP0/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
It looks like SingleTargetP0, P1, and P2 are all the same as SingleTargetApp - both the cs and project.json files.  Is there a reason that you need all of them?